Wangscape/noise/Reseedable.cpp

```diff
@@ -96,7 +92,19 @@ Reseedable Reseedable::operator+(double bias)
     return makeReseedable(result);
 }
 
-Reseedable Reseedable::operator-(Reseedable & other)
+Reseedable & Reseedable::operator+=(Reseedable & other)
```

Comment:
This change comes from one of the commits accidentally introduced during rebase, and was already merged. Also, once again it can't be `const` because the returned `Reseedable` instance has a non-const method that can alter `other`. `other` should be passed by value, but that's fixed in #73.